### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by estebanmino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Rainbow Button
+
+## Demo Dapp
+
+[Demo Link](TBD)
```
Comment (brunobar79) [inline README.md:5]: This should be https://rainbow-me.github.io/rainbow-button

File: README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Rainbow button without styling or custom button
+
+```js
+ReactDOM.render(
+  <RainbowButton
+    chainId={chainId}
+    connectorOptions={connectorOptions}
+    onConnectorInitialized={onConnectorInitialized}
+    render={<button>Custom Rainbow button</button>}
```
Comment (brunobar79) [inline README.md:40]: Shouldn't this be a function that returns a component?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+# Rainbow Button
+
```
Comment (brunobar79) [inline README.md:2]: I think we should have a short description here:

Rainbow Button is a react component that renders an opinionated button built on top of [WalletConnect](https://walletconnect.org) v1 which allows you to connect to [Rainbow](https://rainbow.me) on mobile (via Mobile Deeplinking) and desktop (via QR codes)  and maybe add a recordit gif showing a quick demo

File: README.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+## How to use
+
+```js
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { RainbowButton } from '@rainbow/rainbow-button';
+
+ReactDOM.render(
+  <RainbowButton
+    chainId={chainId}
```
Comment (brunobar79) [inline README.md:24]: I'd pass 1 as the value

File: README.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    connectorOptions={connectorOptions}
```
Comment (brunobar79) [inline README.md:25]: I'd pass an object with the required options instead

File: README.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    onConnectorInitialized={onConnectorInitialized}
```
Comment (brunobar79) [inline README.md:26]: I'd pass a fn logging the connector

File: README.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
+# Rainbow Button
```
Comment (brunobar79) [inline README.md:1]: Mandatory rainbow emoji 🌈

**Review by brunobar79 — APPROVED** [review]
💯🚀👑💪🤝

File: README.md (hunk 7 above)
Comment (estebanmino) [reply README.md:1]: lol

**Pull request merged**